### DISCUSSION TITLE
lx_blog/flutter%E6%8A%80%E6%9C%AF%E7%AF%87/ffi%E5%AE%9E%E6%88%98-%E4%BD%BF%E5%85%AC%E5%8F%B8flutter%E7%A7%BB%E5%8A%A8%E5%BA%94%E7%94%A8%E4%B8%8D%E5%86%8D%E4%BE%9D%E8%B5%96%E5%8E%9F%E7%94%9F%E7%BD%91%E7%BB%9C%E5%BA%93

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#473] [Announcements]
# lx_blog/flutter%E6%8A%80%E6%9C%AF%E7%AF%87/ffi%E5%AE%9E%E6%88%98-%E4%BD%BF%E5%85%AC%E5%8F%B8flutter%E7%A7%BB%E5%8A%A8%E5%BA%94%E7%94%A8%E4%B8%8D%E5%86%8D%E4%BE%9D%E8%B5%96%E5%8E%9F%E7%94%9F%E7%BD%91%E7%BB%9C%E5%BA%93

ffi实战-使公司flutter移动应用不再依赖原生网络库 ffi介绍 Dart 2.5 已经开始，增加了dart:ffi 外部函数接口，用于直接从 Dart 调用 C 代码。 由于原生应用会涉及到C层调用，ffi的产生，就可以实现flutter底层跨平台，摆脱对原生应用原生库的依赖，从而减少flutter和原生项目的交互。 eg: 本文就是将网络请求彻底采用flutter和C实现，获取和加密网络参数不再通过ios或者android的原生网络库获取

http://127.0.0.1:8080/lx_blog/flutter%E6%8A%80%E6%9C%AF%E7%AF%87/ffi%E5%AE%9E%E6%88%98-%E4%BD%BF%E5%85%AC%E5%8F%B8flutter%E7%A7%BB%E5%8A%A8%E5%BA%94%E7%94%A8%E4%B8%8D%E5%86%8D%E4%BE%9D%E8%B5%96%E5%8E%9F%E7%94%9F%E7%BD%91%E7%BB%9C%E5%BA%93.html

**Comment by stdroom** [comment]
订单
